README.md

```diff
@@ -1 +1,34 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is the first test project for Microverse.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
```

Comment:
- [ ] I think it will be good if you add the Getting started section and add information on how to get a local copy. It will make your project professional.